app/Models/Post.php

```diff
@@ -393,4 +393,19 @@ public function hasGoodTag()
 
         return false;
     }
+
+    /**
+    * Scope a query to only return X amount of siblings, determined by limit param.
+    *
+    * @param int $limit
+    * @return \Illuminate\Database\Eloquent\Builder
+    */
+    public function scopeWithPostLimit($query, $limit = null)
+    {
+        return $query->with(['siblings' => function ($query) use ($limit) {
+            if ($limit) {
+                $query->take($limit);
```

Comment:
tried using both `take()` and `limit()` and both seem to return 0 results when there should be siblings :-/